packages/functional/src/__tests__/pipe-test.ts

```diff
@@ -0,0 +1,382 @@
+import { pipe } from '../pipe';
+
+describe('pipe', () => {
+    it('can pipe a single value', () => {
+        expect.assertions(6);
```

Comment:
You only need `expect.assertions()` if the test is `async`. It's designed to let Jest know that one of the awaitables didn't trigger, so the test should fail even though the others might have passed.

Reply:
ohhh did not know that, will remove